generator/generator_test.go

```diff
@@ -335,9 +335,9 @@ cpu:type:x86,ven*fam*mod*:feature:*0081* cbc
 `
 	require.Equal(t, expectedAliases, string(aliasesFile))
 
-	checkFileExistence(t, opts.workDir+"/image.unpacked/usr/lib/firmware/whiteheat.fw.xz")
-	checkFileExistence(t, opts.workDir+"/image.unpacked/usr/lib/firmware/usbdux_firmware.bin.xz")
-	checkFileExistence(t, opts.workDir+"/image.unpacked/usr/lib/firmware/rtw88/rtw8723d_fw.bin.xz")
+	checkFileExistence(t, filepath.Join(opts.workDir, "/image.unpacked/lib/firmware/whiteheat.fw.xz"))
```

Comment:
filepath.Join() should probably be part of another change. Probably in the same commit as 'Converted string concanetation to call to filepath.Join in TestExtraFiles'